e2e/testdata/fn-eval/out-of-place-source-sink/.expected/config.yaml

```diff
@@ -1,3 +1,4 @@
 stdErr: |
   [RUNNING] "gcr.io/kpt-fn/set-namespace:v0.1.3"
-  [PASS] "gcr.io/kpt-fn/set-namespace:v0.1.3"
\ No newline at end of file
+  [PASS] "gcr.io/kpt-fn/set-namespace:v0.1.3"
+  directory "out" doesn't exist, creating the directory...
```

Comment:
Make this consistent with the message for results directory (Present tense, capitalization, blank lines, etc.)

Reply:
This is stale. Updated the logic.